config/ember-try.js

```diff
@@ -101,14 +101,28 @@ module.exports = {
     },
     {
       name: 'ember-data-2.3',
-      dependencies: {
-        "ember": "components/ember#release",
-      },
-      devDependencies: {
-        "ember-data": "~2.3"
+      bower: {
+        dependencies: {
+          "ember": "components/ember#release",
+        },
+        devDependencies: {
+          // Note that the bower dependency for ember-data is set here to ~2.3,
+          // altough since 2.3 ember-data is a proper addon which is installed
+          // via npm; we are setting the version for the bower dependency here
+          // so we make sure that ember-data 2.3 is used. This can be removed
+          // once there is support for removing dependencies in an ember-try
```

Comment:
Let's remove ember-data from the default `bower.json` and add it to each scenario.  That way we do not need ember-try to support removing...